docs: document new filters UI PEM-6202 (#4611)
spectrocloud/librarium: docs/docs-content/clusters/cluster-management/cluster-map-filters.md

---
sidebar_label: "Map and Filter Clusters"
title: "Map and Filter Clusters"
description: "Learn how to view your cluster locations and filter your clusters."
hide_table_of_contents: false
sidebar_position: 180
tags: ["clusters", "cluster management"]
---

The Palette Dashboard provides users with the ability to map and filter Kubernetes clusters. You can find these
capabilities on the **Clusters** page. Mapping and filtering is available for clusters deployed to public clouds, data

> centers and edge hosts.

Reviewer: ```suggestion
centers, and edge hosts.
```

> centers and edge hosts.

Author: Added his change manually since the suggestion is Outdated.

## Cluster List View

Palette provides the **Cluster List View** as an efficient way to view and search your clusters. The cluster list
contains all Palette clusters deployed in the selected Palette scope. The list view also provides filters that allow you
to narrow down clusters that may be dispersed geographically, across multiple regions, or different cloud providers. You
can also search your clusters by name.

### Prerequisites

- A [Palette](https://console.spectrocloud.com) account with the permissions to view and list clusters. Refer to the
  [Roles and Permissions](../../user-management/palette-rbac/project-scope-roles-permissions.md) guide for more
  information.
- One or more active Kubernetes clusters in Palette.

### Enablement

> 1. Login to [Palette](https://console.spectrocloud.com).

Reviewer: ```suggestion
1. Log in to [Palette](https://console.spectrocloud.com).
```

   | --------------- | -------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
   | **Name**        | Filter and display the cluster matching the specified name.                      | Custom string                                                                            |
   | **Environment** | Filter and display clusters based on the cluster type.                           | Configured cluster types                                                                 |
   | **Status**      | Filter and display clusters based on their current status.                       | **Pending**, **Provisioning**, **Running**, **Deleting**, **Importing**, and **Unknown** |
   | **Profiles**    | Filter and display clusters based on the name of the configured cluster profile. | Custom string                                                                            |

   ![Cluster list view](/clusters_cluster-management_cluster-map-filters_cluster-list-view.webp)

> 3. Click on the **Environment** drop-down list and select a cluster type. The cluster list displays matching clusters.

Reviewer: ```suggestion
3. Click on the **Environment** drop-down Menu and select a cluster type. The cluster list displays matching clusters.
```

> 4. Click on the **Status** drop-down list and select the **Running** status. The cluster list displays clusters matching

Reviewer: ```suggestion
4. Click on the **Status** drop-down Menu and select the **Running** status. The cluster list displays clusters matching
```

   | **Environment**     | Filter and display deployed clusters based on the cluster type.                                                                                                                                         | Configured cluster types                                                                 |
   | **Status**          | Filter and display clusters based on their current status.                                                                                                                                              | **Pending**, **Provisioning**, **Running**, **Deleting**, **Importing**, and **Unknown** |
   | **Profiles**        | Filter and display clusters based on the name of the configured cluster profile.                                                                                                                        | Custom string                                                                            |

>    | **Tags**            | Filter and display clusters which have the specified tag. Tags can be specified during cluster deployment or from the **Cluster Settings** tab.                                                         | Custom string                                                                            |

Reviewer: ```suggestion
   | **Tags**            | Filter and display clusters with the specified tag. Tags can be specified during cluster deployment or from the **Cluster Settings** tab.                                                         | Custom string                                                                            |
```

constructed based on the configured cluster location and region. The map view provides filters that allow you to filter
and search through your deployed clusters. You can also search your clusters by name.

### Prerequisites

- A [Palette](https://console.spectrocloud.com) account with the permissions to view and list clusters. Refer to the
  [Roles and Permissions](../../user-management/palette-rbac/project-scope-roles-permissions.md) guide for more
  information.
- One or more active Kubernetes clusters in Palette.

### Enablement

1. Login to [Palette](https://console.spectrocloud.com).

2. Select **Clusters** from the left **Main Menu**. Clusters deployed to data centers or edge hosts need to have their
   locations set manually. Expand the following section to learn how to configure cluster locations. You can skip this
   step for clusters deployed to public clouds.

   <details>

   <summary>Configure Cluster Location</summary>

   Select the cluster to you want to update.

   Click on **Settings**. Then, select **Cluster Settings**. The **Settings** pane appears.

   Select the **Location** option. Search for the cluster location and click on **Save Changes**.

>    Repeat these steps for all the clusters that are missing a location.

Reviewer: Could these be changed to numbered steps?

Just wondering if there was a reason if they weren't.

Author: Added! Thanks for the suggestion!

   | --------------- | -------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
   | **Name**        | Filter and display the cluster matching the specified name.                      | Custom string                                                                            |
   | **Environment** | Filter and display clusters based on the cluster type.                           | Configured cluster types                                                                 |
   | **Status**      | Filter and display clusters based on their current status.                       | **Pending**, **Provisioning**, **Running**, **Deleting**, **Importing**, and **Unknown** |
   | **Profiles**    | Filter and display clusters based on the name of the configured cluster profile. | Custom string                                                                            |

   ![Cluster map view](/clusters_cluster-management_cluster-map-filters_cluster-map-view.webp)

> 4. Click on the **Environment** drop-down list and select a cluster type. The map displays matching clusters.

Reviewer: ```suggestion
4. Click on the **Environment** drop-down Menu and select a cluster type. The map displays matching clusters.
```

> 5. Click on the **Status** drop-down list and select **Running** status. The map displays clusters matching the selected

Reviewer: ```suggestion
5. Click on the **Status** drop-down Menu and select **Running** status. The map displays clusters matching the selected
```
